src/components/home-staking/staking-active.tsx

```diff
@@ -104,7 +105,7 @@ export const StakingActive = forwardRef(
         <Spacer height={20} />
         <Text t8 center i18n={I18N.homeStakingRewards} />
         <Text t3 center color={Color.textGreen1}>
-          {cleanNumber(rewardSum)} ISLM
+          {cleanNumber(formatBalanceWithWEI(rewardSum))} ISLM
```

Comment:
you can create `cleanBalanceBN` function which accepts `Balance` type and makes all formatting in there

Reply:
Migrated to a class solution, please check